README.md

```diff
@@ -7,6 +7,9 @@
 [![Coverage Status](https://coveralls.io/repos/github/octokit/rest.js/badge.svg)](https://coveralls.io/github/octokit/rest.js)
 [![Greenkeeper](https://badges.greenkeeper.io/octokit/rest.js.svg)](https://greenkeeper.io/)
 
+## Installation
+```$ npm install --save @octokit/rest```
```

Comment:
Could you please make it a code block? You’ll have to put the \`\`\` into their own lines.

Remove the `$`, it makes it easier to copy without it. Remove the `--save`, it’s no longer needed.

Reply:
It's better to be explicit here, as some people disable that functionality. Plus it's needed for older versions of npm

Reply:
It’s default behaviour for all supported node/npm versions. People who disable that functionality know what they are doing. Also `--save` is now `--save-prod` as far as I know. I would leave it out.

Reply:
I think I made the changes....